configs/webpack.base.config.js

```diff
@@ -42,7 +45,9 @@ module.exports = {
     extensions: ['.js', '.jsx'],
     alias: {
       Task1: path.resolve(rootFolder, './src/components/task1'),
```

Comment:
Please remove dead code (from task 1) from the codebase.

Reply:
@loginov-rocks fixef